xwrf/postprocess.py

```diff
@@ -205,4 +210,22 @@ def _calc_base_diagnostics(ds: xr.Dataset, drop: bool = True) -> xr.Dataset:
         if drop:
             del ds['PH'], ds['PHB']
 
+    # Earth-relative wind fields (computed according to https://forum.mmm.ucar.edu/threads/how-do-i-convert-model-grid-relative-wind-to-earth-relative-wind-so-that-i-can-compare-model-wind-to-observations.179/)
+    if {'U', 'V', 'SINALPHA', 'COSALPHA'}.issubset(ds.data_vars):
+        u_model, v_model = _destag_variable(ds['U']), _destag_variable(ds['V'])
```

Comment:
```suggestion
        u_model, v_model = _destag_variable(ds['U'].variable), _destag_variable(ds['V'].variable)
```

While I avoided adding a type check to `_destag_variable` since it was a private method, given this and https://github.com/xarray-contrib/xwrf/issues/99#issuecomment-1249792542, would it be worth adding such a check (i.e., raise ValueError if `datavar` is not an instance of `xr.Variable`)?

Reply:
Yes, I'd say so. If it is so subtle but really makes a difference then I think type-checking is very much in order.